### PR TITLE
Cn video url fix

### PR DESCRIPTION
**Pull request by wjllance**
I add cn_url field in utils.coffee
and fix both teacher side and student side video display problem

### CODE REVIEW

File: app/views/courses/CourseVideosComponent.vue [hunk 1]
```diff
@@ -53,6 +54,14 @@ export default Vue.extend({
       type: String,
       default: null
     }
+  },
+  data: () => ({
+	  videoUrls: []
```
Comment (AndrewJakubowicz) [inline app/views/courses/CourseVideosComponent.vue]: Can we make this an object with key value pairs?
Pushing onto an array makes everything dependent on the index. This means we lose the information of the order.

**Comment by wjllance** [comment]
> Looks great. Just need a data structure change in `CourseVideosComponent.vue`.
> Can we use an object with properties for the different videos. Currently the videos depend on a specific order which means that information is lost. I.e. It's hard to understand which video `videoUrls[0]` is.
> 
> Instead could you please make it so that in the template it's `videoUrls['basic_syntax']`, `videosUrls['while_loops']`, and `videosUrls['variables']`?

updated

**Comment by AndrewJakubowicz** [comment]
I manually tested this on the teacher dashboard.

The china videos have two issues.
Because they don't use Vimeo they throw an error:
`Error: "The player element passed isn’t a Vimeo embed."`

They also don't pause when you close the modal. Didn't check for students.

**Comment by wjllance** [comment]
> Make sure user experience doesn't degrade for China videos.

So I need to replace vimeo player code with some other methods for China

**Comment by wjllance** [comment]
I seems did something bad for this branch...
https://github.com/codecombat/codecombat/pull/5235

**Comment by AndrewJakubowicz** [comment]
Closing as changes are now in the new #5235 PR.

**Pull request closed without merging**